invenio_app_rdm/utils/checks.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+#
+# Invenio-Checks is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Checks utilities."""
+
+from flask import current_app, g
+from invenio_communities.proxies import current_communities
+from invenio_rdm_records.requests import CommunityInclusion, CommunitySubmission
+from sqlalchemy import case
+
+
+def resolve_checks(record_uuid, request, community=None):
+    """Resolve the checks for this draft/record related to the community and the request."""
+    # Early exit if checks are not enabled.
+    enabled = current_app.config.get("CHECKS_ENABLED", False)
+
+    if not enabled:
+        return None
+
+    # Early exit if not draft submission nor record inclusion
+    request_type = request["type"]
+    is_draft_submission = request_type == CommunitySubmission.type_id
```

Comment:
```suggestion
    is_draft_review = request_type == CommunitySubmission.type_id
```
for naming consistency